beestation.dme

```diff
@@ -2480,7 +2482,6 @@
 #include "code\modules\mob\living\carbon\human\species_types\android.dm"
 #include "code\modules\mob\living\carbon\human\species_types\apid.dm"
 #include "code\modules\mob\living\carbon\human\species_types\corporate.dm"
-#include "code\modules\mob\living\carbon\human\species_types\debug.dm"
```

Comment:
Why?

Reply:
Sprite Sheets was killed, this was used to debug it.